x/observer/utils/utils.go

```diff
@@ -0,0 +1,19 @@
+package utils
+
+import (
+	"gitlab.com/thorchain/tss/go-tss/blame"
+
+	"github.com/zeta-chain/zetacore/x/observer/types"
+)
+
+func ConvertNodes(n []blame.Node) (nodes []*types.Node) {
```

Comment:
can we move this to the client directory instead?  or under `pkg` 
we do not have a separate utils directory in any other module, it would be good to keep the module lever folder structure consistent

Reply:
Ok updated